src/bindings/python/src/pyopenvino/core/async_infer_queue.cpp

```diff
@@ -163,27 +182,117 @@ void regclass_AsyncInferQueue(py::module m) {
             }
         },
         py::arg("inputs"),
-        py::arg("userdata"));
+        py::arg("userdata"),
+        R"(
+            Run asynchronous inference using next available InferRequest.
+
+            Parameters
+            ----------
+            inputs : dict[Union[int, str, openvino.runtime.ConstOutput] : openvino.runtime.Tensor]
+                Data to set on input tensors of next available InferRequest from
+                AsyncInferQueue's pool.
+
+            userdata : Any
+                Any data that will be passed to a callback.
+
+            Returns
+            ----------
+            start_async : None
+        )");
+
+    cls.def(
+        "is_ready",
+        [](AsyncInferQueue& self) {
+            return self._is_ready();
+        },
+        R"(
+            One of 'flow control' functions. Blocking call.
+            If there is at least one free InferRequest in a pool, returns True. 
```

Comment:
It blocks while there is no free request in the pool and after that it either returns True either throws exception from py callback, so maybe we should change description that it just blocks while no request will be available.

Reply:
Ok, will extend that comment.

Reply:
Discussed offline